scripts/utils.ts

```diff
@@ -0,0 +1,109 @@
+import fs from "fs";
+
+import dotenv from "dotenv";
+dotenv.config();
+
+export const infuraKey =
+  process.env.INFURA_KEY || "84842078b09946638c03157f83405213";
+
+export const factoryDeployer: string =
+  "0x4200000000000000000000000000000000000012";
+
+export const filePath = "./projects/standard-bridge/";
+
+export const network = {
+  MAINNET: 1,
+  OP_MAINNET: 10,
+  KOVAN: 42,
+  OP_KOVAN: 69
+};
```

Comment:
nit: if you add an `as const` to stuff like this it makes it appear as a constant when you hover over it in vscode